stubs/requests/METADATA.toml

```diff
@@ -1,2 +1,3 @@
 version = "2.27.*"
+requires = ["types-urllib3<1.27"]
```

Comment:
Why not 1.27 or newer?

Reply:
This echoes the requests's `setup.py`. I'm not sure about urllib3's stability guarantees.

Reply:
Maybe we should add a comment.